src/main.ts

```diff
@@ -82,6 +83,34 @@ export async function activate(
         requiredPackageIds.push('OmniSharp');
     }
 
+    const dotnetRuntimeExtensionId = 'ms-dotnettools.vscode-dotnet-runtime';
+    const requiredDotnetRuntimeExtensionVersion = '2.2.3';
+
+    const dotnetRuntimeExtension = vscode.extensions.getExtension(dotnetRuntimeExtensionId);
```

Comment:
Note: Our extension requires vscode 1.81.1 or higher. Looks like you already are at ^1.93.0 though so its good.